lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -319,8 +321,7 @@ <h4 class="message-title">
     </p>
   </section>
   % endif
-</article>
-</article>
```

Comment:
It appears to be properly closed.  I did not find any orphan ```article``` elements in this template or on the rendered page